webapp/src/views/projects/import/ImportView.tsx

```diff
@@ -123,6 +123,20 @@ export const ImportView: FunctionComponent = () => {
             addFilesMutation={dataHelper.addFilesMutation}
           />
         ))}
+        {dataHelper.addFilesMutation.data?.warnings.map((item) => (
+          <Box key={item.code} mt={4} data-cy="import-file-warnings">
+            <Alert severity="warning">
+              <AlertTitle>
+                <TranslatedError
+                  code={'import_file_warning_header_' + item.code}
```

Comment:
Can you just pass `code`? Why the concatenation?

Reply:
There is the header and the message. Or maybe I could use just the message and omit the header.

Reply:
Created a new component for translated warning box.